docs/manual/docs/cmd/spo/tenant/tenant-recyclebinitem-remove.md

```diff
@@ -0,0 +1,40 @@
+# spo tenant recyclebinitem remove
+
+Removes the specified deleted Site Collection from Tenant Recycle Bin
```

Comment:
We should change `Site Collection` to `site collection` and `Tenant Recycle Bin` to `tenant recycle bin`